packages/contracts/src/EmailSigner.sol

```diff
@@ -70,13 +70,14 @@ contract EmailSigner is IERC1271, Initializable {
         address _verifierAddr,
         uint256 _templateId
     ) public initializer {
-        accountSalt = _accountSalt;
-        templateId = _templateId;
-
+        if (_accountSalt == bytes32(0)) revert InvalidAccountSalt();
```

Comment:
`InvalidAccountSalt` & `InvalidTemplateId` reverts are not unit tested. Doesn't need to hold up PR, but if you don't add them can you add and link a linear issue?